src/arns.lua

```diff
@@ -453,7 +486,8 @@ end
 ---   - name string The name of the record
 ---   - intent string The intended action type (Buy-Record/Extend-Lease/Increase-Undername-Limit/Upgrade-Name)
 ---   - currentTimestamp number The current timestamp
---- @return number The token cost in mIO of the intended action
+---   - from string|nil The target address of the intended action
+--- @return TokenCostResult The token cost in mIO of the intended action
 function arns.getTokenCost(intendedAction)
```

Comment:
I kinda wonder if we should rename this now that it returns more than just a cost. It's creating a little bit of readability friction in the code.

Reply:
This commit just seems to update docs.